index.html

```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width">
-  <title>no title</title>
```

Comment:
Here please provide a better title for the document

Reply:
I am just testing how to give a code review

Reply:
Great job reviewing the project 👍. But, I think it will be better if you give an explanation on why the required changes are needed so that the student will know why they are doing it. 😊 🛠️